eessi_bot_job_manager.py

```diff
@@ -533,6 +561,33 @@ def process_finished_job(self, finished_job):
         comment_update = f"\n|{dt.strftime('%b %d %X %Z %Y')}|finished|"
         comment_update += f"{comment_description}|"
 
+        # check if _bot_jobJOBID.test exits
+        # TODO if not found, assume test was not run (or failed, or ...) and add
+        # a message noting that ('not tested' + 'test suite not run or failed')
+        # --> bot/test.sh and bot/check-test.sh scripts are run in job script used by bot for 'build' action
+        job_test_file = f"_bot_job{job_id}.test"
+        job_test_file_path = os.path.join(new_symlink, job_test_file)
+        job_tests = self.read_job_test(job_test_file_path)
+
+        job_test_unknown_fmt = finished_job_comments_cfg[JOB_TEST_UNKNOWN_FMT]
+        # set fallback comment_description in case no test file was found
+        # (self.read_job_result returned None)
+        comment_description = job_test_unknown_fmt.format(filename=job_test_file)
+        if job_tests:
+            # get preformatted comment_description or use previously set default for unknown
+            comment_description = job_tests.get(JOB_TEST_COMMENT_DESCRIPTION, comment_description)
+
+        # report to log
+        log(f"{fn}(): finished job {job_id}, test suite result\n"
+            f"########\n"
+            f"comment_description: {comment_description}\n"
+            f"########\n", self.logfile)
+
+        dt = datetime.now(timezone.utc)
+
+        comment_update += f"\n|{dt.strftime('%b %d %X %Z %Y')}|tested|"
```

Comment:
I would use "`test result`" rather than "`testing`"

```suggestion
        comment_update += f"\n|{dt.strftime('%b %d %X %Z %Y')}|test result|"
```

which changes the status overview in the PR comment from:

|date|job status|comment|
|----------|----------|------------------------|
|...|...|...|
|Nov 26 13:39:10 UTC 2023|tested|some checks FAILED in test phase|


to

|date|job status|comment|
|----------|----------|------------------------|
|...|...|...|
|Nov 26 13:39:10 UTC 2023|test result|some checks FAILED in test phase|